guides/common/modules/con_defining-content-access-strategies-for-hosts.adoc

```diff
@@ -0,0 +1,106 @@
+[id="defining-content-access-strategies-for-hosts_{context}"]
+= Defining content access strategies for hosts
+
+When defining your content lifecycle in {Project}, you can use content views and lifecycle environments to make content, such as errata and packages, available only to certain hosts.
+This enables you to restrict the content your hosts have access to.
+
+Implement your content access strategies in the following order:
+
+. _Define content views and lifecycle environments_.
```

Comment:
that is quite unique to have a whole sentence in italic. maybe use "glossary items like this ::" instead?